internal/daysteps/daysteps.go

```diff
@@ -0,0 +1,87 @@
+package daysteps
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/spentcalories"
+)
+
+const (
+	// Длина одного шага в метрах
+	stepLength = 0.65
+	// Количество метров в одном километре
+	mInKm = 1000
+)
+
+//Алгоритм реализации функции:
+
+func parsePackage(data string) (int, time.Duration, error) {
+	//Разделить строку на слайс строк.
+	//Проверить, чтобы длина слайса была равна 2, так как в строке данных у нас количество шагов и продолжительность.
+	parsedData := strings.Split(data, ",")
+	if len(parsedData) != 2 {
+		return 0, 0, fmt.Errorf("incorrect Input")
+	}
+
+	// Преобразовать первый элемент слайса (количество шагов) в тип int. Обработать возможные ошибки.
+	// При их возникновении из функции вернуть 0 шагов, 0 продолжительность и ошибку.
+	// Проверить: количество шагов должно быть больше 0. Если это не так, вернуть нули и ошибку.
+	steps, err := strconv.Atoi(parsedData[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше ошибки оборачивать
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```